src/pages/storage/actions/CustomIsoBtn.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from "react";
+import { Button, Icon } from "@canonical/react-components";
+import { useNavigate } from "react-router-dom";
+
+interface Props {
+  project: string;
+}
+
+const CustomIsoBtn: FC<Props> = ({ project }) => {
+  const navigate = useNavigate();
+  const href = `/ui/project/${project}/storage/custom-isos`;
+
+  return (
+    <Button
+      className="u-no-margin--bottom u-float-right"
+      hasIcon
+      href={href}
+      onClick={(e) => {
```

Comment:
just a nit, I think it's better to define event handlers outside of the return to avoid mixture of JS and html code as much as possible